horace_core/sqw/coord_transform/@projection/private/get_contributing_pix_ind_.m

```diff
@@ -20,5 +23,9 @@
 % get good indices (including integration axes and plot axes with only one bin)
 % roun-off them to 7 significant digit to eliminate round-off errors,
 % generated by epsilon around the urange border
-indx=round(indx(ok,:),7);
+
+indx  = indx(ok,:);
+%indx=round(indx(ok,:),7); %-- very slow. Below seems much faster. Consider
```

Comment:
This strikes me as bizarre. One would think MATLAB would implement the optimal round for their system.

Reply:
yes -- I was extremely surprised by this discovery. It works like its done through printing to strings and reading back.